versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -77,22 +93,178 @@ spec:
+
+      analyzer:
+        description: >
+          Name of the analyzer to use for indexing. Unless search_analyzer is
+          specified this analyzer is used for both indexing and searching.
+          Only valid for 'type: text'.
+        type: string
+
+      search_analyzer:
+        description: >
+          Name of the analyzer to use for searching. Only valid for 'type: text'.
+        type: string
+
+      multi_fields:
+        description: >
+          It is often useful to index the same field in different ways for
+          different purposes. This is the purpose of multi-fields. For instance,
+          a string field could be mapped as a text field for full-text search,
+          and as a keyword field for sorting or aggregations.
+
+          Only honored for 'type: text/keyword/wildcard'.
+        $ref: "#" # JSON-schema syntax for pointing to the root of the schema
+
+      # TODO: Fleet only honors this for 'type: wildcard'. That seems wrong.
+      null_value:
+        description: >
+          The null_value parameter allows you to replace explicit null values
+          with the specified value so that it can be indexed and searched.
+
+          A null value cannot be indexed or searched. When a field is set to
+          null, (or an empty array or an array of null values) it is treated as
+          though that field has no values.
+
+          The null_value needs to be the same data type as the field. For
+          instance, a long field cannot have a string null_value.
+
+          The null_value only influences how data is indexed, it doesn’t modify
+          the _source document.
+        examples:
+          - "NULL"
+
+      ignore_above:
+        description: >
+          Strings longer than the ignore_above setting will not be indexed or
+          stored. For arrays of strings, ignore_above will be applied for each
+          array element separately and string elements longer than ignore_above
+          will not be indexed or stored. Only honored for
+          'type: keyword/wildcard'. Defaults to 1024.
+        type: integer
+        default: 1024
+
+      object_type:
+        description: >
+          Required for `type: array` to specify the data type in the array.
+        type: string
+
+      path:
+        description: >
+          For alias type fields this is the path to the target field. Note that
+          this must be the full path, including any parent objects
+          (e.g. object1.object2.field).
+        type: string
+
+      normalizer:
+        description: >
+          Specifies the name of a normalizer to apply to keyword fields. A
+          simple normalizer called lowercase ships with elasticsearch and can be
+          used. Custom normalizers can be defined as part of analysis index
+          settings.
+        type: string
+
+      # TODO: This two are read by Kibana, but AFAICT they have no use in
+      # Elasticsearch. And nothing in elastic/integrations uses it.
```

Comment:
Then maybe we shouldn't add them by now?

Reply:
I would be precautious about them too unless we're sure that Fleet fully supports them. Is there a requirement now to allow for and use all these properties?

Reply:
I'll remove them and make a note to follow up to determine why these exist in Fleet. From there we can determine the next steps - either add them here or remove them from Fleet.